x11-terms/kitty/kitty-0.15.0.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit python-single-r1 toolchain-funcs xdg
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/kovidgoyal/kitty.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/kovidgoyal/kitty/releases/download/v${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="A modern, hackable, featureful, OpenGL-based terminal emulator"
+HOMEPAGE="https://github.com/kovidgoyal/kitty"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="debug doc imagemagick wayland"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+COMMON_DEPS="
+	${PYTHON_DEPS}
+	>=media-libs/harfbuzz-1.5.0:=
+	sys-apps/dbus
+	sys-libs/zlib
+	media-libs/libcanberra
+	media-libs/libpng:0=
+	media-libs/freetype:2
+	media-libs/fontconfig
+	x11-libs/libXcursor
+	x11-libs/libXrandr
+	x11-libs/libXi
+	x11-libs/libXinerama
+	x11-libs/libxkbcommon[X]
+	x11-libs/libxcb[xkb]
+	wayland? (
+		dev-libs/wayland
+		>=dev-libs/wayland-protocols-1.17
+	)
+"
+RDEPEND="
+	${COMMON_DEPS}
+	imagemagick? ( virtual/imagemagick-tools )
+"
+DEPEND="${RDEPEND}
+	media-libs/mesa[X(+)]
+	sys-libs/ncurses
+"
+[[ ${PV} == *9999 ]] && DEPEND+=" >=dev-python/sphinx-1.7[${PYTHON_USEDEP}]"
```

Comment:
Are sphinx's modules really used here, or is it just executed to build doc files? Also does this still hold truth with 9999 only? 

What I'm getting at is if it's just executed, you can drop PYTHON_USEDEP and put this into BDEPEND.